src/main/java/tool/updater/UpdateController.java

```diff
@@ -104,14 +105,22 @@ public static void check4update(boolean bInformationOnly) {
 	}
 
 	public static String getHOupdateURL(String full_version, double version, String versionType) {
+		String suffix = ".zip";
+		if (isMac()){
+			suffix = "_OSX" + suffix;
+		}
 		if (versionType == "DEV") {
-			return "https://github.com/akasolace/HO/releases/download/dev/HO_" + full_version + ".zip";
+			return "https://github.com/akasolace/HO/releases/download/dev/HO_" + full_version + suffix;
 		} else {
 			String ver = Double.toString(version);
-			return "https://github.com/akasolace/HO/releases/download/" + ver + "/HO_" + ver + ".zip";
+			return "https://github.com/akasolace/HO/releases/download/" + ver + "/HO_" + ver + suffix;
 		}
 	}
 
+	final public static boolean isMac() {
```

Comment:
a isMac function already exists, could you please use it instead of redefining a new one